schema/deploy/mutations/discard_additional_funding_source_form_change.sql

```diff
@@ -0,0 +1,36 @@
+-- Deploy cif:mutations/discard_additional_funding_source_form_change to pg
+-- requires: tables/form_change
+-- requires: tables/project_revision
+
+begin;
+
+create or replace function cif.discard_additional_funding_source_form_change(revision_id int, source_index int)
+returns setof cif.form_change
+as $discard_additional_funding_source_form_change$
+declare
+form_change_record record;
+
+begin
+  for form_change_record in select * from cif.form_change
```

Comment:
I'm confused by the loop here. Shouldn't we only ever have one record being deleted by this function (unique on revision_id and source_index)?

Reply:
Can't we just update/delete based on a form_change id since each additional_funding_source is (currently) its own form_change? That would solve the delete by index bug I described

Reply:
True 💯! Thanks for raising that Dylan.

Reply:
@dleard Do we need to do the same for `discard_funding_parameter_form_change` ?!

Reply:
I would say so if @dleard agrees!?